### DISCUSSION TITLE
New GitBook Styling - How to benefit from it?

### DISCUSSION THREAD

**Discussion by arnaudbesnier** [GitbookIO/community#501] [2. Published content] [answered]
I was reading your documentation and none of the block elements look like our own documentation.

For instance hints look like this, in our documentation:
![Screenshot 2024-03-01 at 16 00 26](https://github.com/GitbookIO/community/assets/664261/523cc2f1-1659-49c0-8175-cc0ba307781f)

When, it looks like this, in [GitBook documentation](https://docs.gitbook.com/content-editor/blocks/hint#example-of-a-hint):
![Screenshot 2024-03-01 at 16 00 46](https://github.com/GitbookIO/community/assets/664261/7df5d42c-9e07-456c-974e-d89360979f96)

How can we benefit from this new, lean style?
I found nothing in our GitBook settings.

Thanks for your help 🙏

**Comment by addisonschultz** [comment, marked as answer]
@arnaudbesnier great catch! We're working on updating the way we render content in GitBook at the moment, including major updates to the styling of the blocks that render in GitBook.

You should be able to see what it looks like for you, by adding ~open in the url. For instance, if your published site is at `https://open-source.gitbook.io/shot-scraper`, you can preview the updates coming soon at `https://open-source.gitbook.io/~open/shot-scraper`

let me know if that makes sense!

**Comment by kacperholak** [comment]
Has the new styling gone live yet? We can see some random changes in our documentation, suggesting there are some issues:
![image](https://github.com/GitbookIO/community/assets/96750757/794b2486-d7ab-4e68-99c8-9bd5ec48ef47)

**Reply by addisonschultz** [reply]
Hey @kacperholak - This looks like a regression, I've filed a ticket for us to fix this! That being said, the new changes should be coming very shortly, which seems to already have fixed the issue, so hang tight!

**Reply by ahcsco** [reply]
What happened to new changes? @addisonschultz

**Reply by addisonschultz** [reply]
@shimshim938 we've got all of the updates listed in our changelog! https://changelog.gitbook.com/march-2024/4-march-huge-improvements-to-your-published-documentation

**Comment by aikoii** [comment]
Hey all 👋! We’ve redesigned our hint blocks for a cleaner, more consistent look across all docs. Hope you like the new design 😊!

Editor: 

![CleanShot 2024-12-05 at 16 05 10@2x](https://github.com/user-attachments/assets/573c74c6-5106-47c2-9e5f-853c26e08444)


Published site:

![CleanShot 2024-12-05 at 16 05 20@2x](https://github.com/user-attachments/assets/5bdfcd66-599d-4b35-aaf9-21f996099d7d)
